app.py

```
@@ -1,67 +1,72 @@
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)